src/v0/destinations/monday/util.js

```
@@ -62,7 +82,16 @@ const getColumnId = (columnTitle, board) => {
  * @returns
  */
 const getColumnValue = (properties, columnName, key, board) => {
+  if (!board?.boards?.[0]) {
+    throw new ConfigurationError(BOARD_DATA_INVALID_ERROR);
+  }
+  
   const { columns } = board.boards[0];
+  
+  if (!columns || !Array.isArray(columns)) {
```

Should we throw an error if columns is not of type array?